### ISSUE TITLE
Support Msbuild 12

### ISSUE THREAD

**Issue by akashlal** [p-org/PSharp#236]
`PSharpSyntaxRewriter` should emit `"foo"` instead of `nameof(foo)`. This should be all that is required to support `msbuild 12.0`.

**Comment by TedHartMS** [comment]
Resolved in acf46a7. To activate in MSBuild, add the CSharpVersion to the Rewriter task in the .targets file:
```
    <Rewriter
        InputFiles="@(PSharp)"
        _CSharpVersion="5.0"_
        OutputFiles="@(PSharp->'$(IntermediateOutputPath)%(FileName)%(Extension).cs')">
      <Output TaskParameter="OutputFiles" ItemName="Compile" />
    </Rewriter>
```

**Comment by TedHartMS** [comment]
Use LangVersion setting so users don't have to modify NuGet-installed files
